bigbluebutton-tests/playwright/README.md

```
@@ -5,28 +5,34 @@ Tests for BigBlueButton using Playwright.
 ## Setup (with an existing BigBlueButton server)
 
 You need to install the dependencies:
+
 ```bash
 $ cd ../bigbluebutton-tests/playwright
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)